metrics.md

```diff
@@ -0,0 +1,228 @@
+> * **Price Provider**: A price provider is a service that provides price data for a given market. For example, the Coinbase API is a price provider for the Coinbase markets.
+> * **Market Map Provider**: A market map provider is a service that supplies the markets that the side-car needs to fetch data for.
+
+Slinky exposes metrics on the `/metrics` endpoint on port `8002` by default. These metrics are in the Prometheus format and can be scraped by Prometheus or any other monitoring system that supports Prometheus format.
+
+## Health Metrics
+
+There are three primary health metrics that are exposed by Slinky:
+
+* [`side_car_health_check_system_updates_total`](#side_car_health_check_system_updates_total): This metric is a counter that increments every time the side-car updates its internal state. This is a good indicator of the side-car's overall health.
+* [`side_car_health_check_ticker_updates_total`](#side_car_health_check_ticker_updates_total): This metric is a counter that increments every time the side-car updates the price of a given market. This is a good indicator of the overall health of a given market.
+* [`side_car_health_check_provider_updates_total`](#side_car_health_check_provider_updates_total): This metric is a counter that increments everytime the side-car utilizes a given providers market data. This is a good indicator of the health of a given provider. Note that providers may not be responsible for every market. However, the side-car correctly tracks the number of expected updates for each provider.
+
+### `side_car_health_check_system_updates_total`
+
+This metric should be increasing. Specifically, the rate of this metric should be inversely correlated to the configured `UpdateInterval` in the oracle side-car configuration (`oracle.json`). To check this, you can run the following query in Prometheus:
+
+```promql
+rate(side_car_health_check_system_updates_total[5m])
+```
+
+![Architecture Overview](./resources/side_car_health_check_system_updates_total_rate.png)
+
+
+### `side_car_health_check_ticker_updates_total`
+
+This should be a monotonically increasing counter for each market. Each market's counter should be relatively close to the `side_car_health_check_system_updates_total` counter.
+
+To verify that the rate of updates for each market is as expected, you can run the following query in Prometheus:
+
+```promql
+rate(side_car_health_check_ticker_updates_total[5m])
+```
+
+![Architecture Overview](./resources/side_car_health_check_ticker_updates_total_rate.png)
+
+### `side_car_health_check_provider_updates_total`
+
+This metric should be monotonically increasing for each (provider, market) pair. To verify that the rate of updates for each provider is as expected, you can run the following query in Prometheus:
+
+```promql
+rate(side_car_health_check_provider_updates_total{provider="coinbase_api", success="true"}[5m])
```

Comment:
I'm a little confused about why the success rate over 5m is like 2/3 for all of the providers in these examples.

Reply:
I'm going to update so that we use the rate described above and follow up with explanations on the examples.